roles/template-configs/tasks/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+- name: Template Browbeat configuration
+  template:
+    "src={{ browbeat_config_file }} \
+     dest={{ ansible_env.HOME }}/browbeat/browbeat-config.yaml"
```

Comment:
Is it better to use lookup? Do we know if there are any specific advantages/disadvantages to using lookups? Not a hard requirement, just curious.

Reply:
Not even sure what you mean by lookup here.

Reply:
Something like https://github.com/smalleni/infrared-scale/blob/master/ansible/deploy-scale-cloud.yml#L10